frontend/lib/identification/card_detail_view.dart

```diff
@@ -1,45 +1,40 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
-import 'package:flutter_svg/svg.dart';
 import 'package:graphql_flutter/graphql_flutter.dart';
-import 'package:intl/intl.dart';
 
 import '../graphql/graphql_api.dart';
+import 'card/eak_card.dart';
+import 'card/id_card.dart';
 import 'card_details.dart';
-import 'id_card.dart';
 import 'verification_qr_code_view.dart';
 
 class CardDetailView extends StatelessWidget {
   final CardDetails cardDetails;
-  final VoidCallback onOpenQrScanner;
+  final VoidCallback openQrScanner;
+  final VoidCallback startVerification;
 
-  CardDetailView({Key key, this.cardDetails, this.onOpenQrScanner})
+  CardDetailView(
+      {Key key, this.cardDetails, this.openQrScanner, this.startVerification})
       : super(key: key);
 
-  get _formattedExpirationDate => cardDetails.expirationDate != null
-      ? DateFormat('dd.MM.yyyy').format(cardDetails.expirationDate)
-      : "unbegrenzt";
-
   @override
   Widget build(BuildContext context) {
-    final getRegions = GetRegionsQuery();
+    final regionsQuery = GetRegionsByIdQuery(
+        variables: GetRegionsByIdArguments(
+            ids: IdsParamsInput(ids: [cardDetails.regionId])));
+
     return Query(
         options: QueryOptions(
-            document: getRegions.document,
-            variables: getRegions.getVariablesMap()),
-        builder: (result, {fetchMore, refetch}) {
+            document: regionsQuery.document,
+            variables: regionsQuery.getVariablesMap()),
+        builder: (result, {refetch, fetchMore}) {
+          var region = result.isConcrete
+              ? regionsQuery.parse(result.data).regionsById[0]
```

Comment:
```suggestion
              ? regionsQuery.parse(result.data)?.regionsById?.first
```
Hm, I hoped this would make it fail-safe, but it still crashes when I try to add a card (in `parse` already). Never mind, once the BE is deployed it will just work anyways.

Reply:
Hmm... strange... I thought `result.isConcreate` is only true, if there is no graphql error :thinking: (or why else should `.parse` throw? :thinking:

Reply:
I do not have the error message right now, but it was something like "tried to call `[]("regions did")` on null"